source/stylesheets/_headings.scss

```diff
@@ -1,15 +1,17 @@
-h2.underlined {
-  color: rgba(101, 87, 87, 0.67);
-  margin-bottom: 1em;
+.underlined-heading {
+  color: rgb(183, 183, 183);
```

Comment:
My professional web development is +pre css preprocessors_, but I assume we can have variables, right? Should we have all the colors somewhere, in like `common.scss` or `colors.scss`?